CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## 4.0.0-RC1 / 2016-06-21
 
+* [GeneratorBundle] change entity generator to use the same fields as pagepart generator [#670](https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/670) ([@treeleaf](https://github.com/treeleaf))
```

Comment:
Can you remove the changes form the CHANGELOG, it will be generated automatically